src/wifi_interface.rs

```diff
@@ -474,6 +474,7 @@ impl<'s, 'n: 's> Socket<'s, 'n> {
                     .network_interface()
                     .get_socket_and_context::<TcpSocket>(self.socket_handle);
                 let remote_endpoint = (addr, port);
+                sock.set_ack_delay(Some(smoltcp::time::Duration::from_millis(100)));
```

Comment:
Is this something that should only be used with the `ps_min_modem` feature, or just a general improvement?

Reply:
I think it shouldn't hurt in any case - the default 10ms are also somewhat optimistic but not really sure t.b.h